### ISSUE TITLE
R-Instat run as  administrator? 

### ISSUE THREAD

**Issue by N-thony** [IDEMSInternational/R-Instat#7502]
During the KMD workshop with version 0.7.5 we found that `corrr` package was missing in the use of the `Correlation Dialogue` and needed to be installed, we could easily install the package through `Tools > Install R Packages...` but we couldn't since the system required to have the administrator grant to install any or make a changes in disk `C:`, Roger was stuck we this situation since the only thing to do was to close R-Instat since by defaut it run and run it again as administrator i.e right click on R-Instat icon, and then run as administrator in the menu context. Could we force R-Instat by default to always run as administrator.
@ChrisMarsh82 @dannyparsons @rdstern what do you think?

**Comment by rdstern** [comment]
@N-thony I agree we don't want to have to run R-Instat as an administrator.  But I would also like anyone to be able to add packages, without needing administrator rights.  This is what I can do when using RStudio.  So I am not sure that the packages - at least extra packages need to be in the directories where administrator rights are needed.  @ChrisMarsh82 I asked this in #7064 in December last year, with a follow-up query in February this year.  I still wonder!
